code/modules/modular_computers/file_system/programs/techweb.dm

```diff
@@ -7,8 +7,8 @@
 	size = 16
 	tgui_id = "NtosTechweb"
 	program_icon = "atom"
-	required_access = ACCESS_HEADS
-	transfer_access = ACCESS_RD
+	required_access = list(ACCESS_HEADS, ACCESS_RESEARCH)
+	transfer_access = list(ACCESS_RESEARCH)
```

Comment:
should keep this as RD-only and make scientists spawn with tablets that have the app. RD is still expected to supervise research so giving them a way to remove certain people from researching would be nice.